crates/anvil/src/config.rs

```diff
@@ -1171,9 +1186,10 @@ latest block number: {latest_block}"
                 // the next block
                 let next_block_base_fee = fees.get_next_block_base_fee_per_gas(
                     block.header.gas_used,
-                    block.header.gas_limit,
+                    gas_limit,
```

Comment:
not sure about this

Reply:
this should be okay, but in case u128 the basefee would be more or less locked to the lowest possible value 7.
I assume this is okay if you disable the gas limit